manifests/charts/global.yaml

```diff
@@ -39,6 +39,8 @@ global:
 
   # One central istiod controls all remote clusters: disabled by default
   centralIstiod: false
```

Comment:
can you add a deprecation message? https://github.com/istio/istio/blob/02863894d52871e76364b5ae88697b2710d401eb/operator/pkg/apis/istio/v1alpha1/validation/validation.go#L80

And indicate on the comments this is deprecated in favor of `externalIstiod`

Reply:
fixed

Reply:
added

Reply:
@irisdingbj I am going to approve this PR, but would like you to follow up with another PR that adds the validation to istio/operator/pkg/apis/istio/v1alpha1/validation/validation.go.